src/main/environment/common_ci.properties

```diff
@@ -64,7 +64,17 @@ labWL=7
 radioWL=7
 oncoWL=7
 tcSpeclistWL=7
-
+servicePointID: @env.SERVICE_POINT_ID@
+parkingPlaceID: @env.PARKING_PLACE_ID@
+providerServiceMapID: @env.PROVIDER_SERVICE_MAP_ID@
+vanID: @env.VAN_ID@
+serviceID: @env.SERVICE_ID@
+providerID: @env.PROVIDER_ID@
+appId: @env.APP_ID@
+authKey: @env.AUTH_KEY@
```

Comment:
@sac2kadam can we add a context to this?
What is this AUTH keys about?
@ravishanigarapu can you help?

Reply:
@devikasuresh20 and   @srishtigrp78 
they are checking about AUTH key,

Reply:
As per my analysis till now, we are taking these properties from application.properties file to get nurse master data for beneficiary in code using state id, it is being used in PatientAppCommonMasterController.
@drtechie @sac2kadam